LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Sonja Heinze
+
```

Comment:
Thanks! I should have added a license months ago. I'd like to add 3Rafal here as well. We can leave it like this for now, and I ask Rafal when he's back if I can add him.